src/OpenLoco/Windows/PromptOkCancelWindow.cpp

```diff
@@ -97,6 +110,17 @@ namespace OpenLoco::Ui::Windows
         return _result != 0;
     }
 
+    // 0x00447125
+    void promptOkCancelInput(uint32_t charCode, uint32_t keyCode)
+    {
+        auto window = WindowManager::find(WindowType::confirmationPrompt);
+        if (window == nullptr)
+            return;
+
+        if (charCode == VK_ESCAPE)
```

Comment:
We should really switch to the SDL2 constants at some point. I'm not sure how feasible it is at this point yet, though. Not something for this PR, at any rate.